builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -261,6 +287,10 @@ func resourceAwsElasticacheClusterRead(d *schema.ResourceData, meta interface{})
 		d.Set("security_group_ids", c.SecurityGroups)
 		d.Set("parameter_group_name", c.CacheParameterGroup)
 		d.Set("maintenance_window", c.PreferredMaintenanceWindow)
+		log.Printf("[INFO] Found %s as the Snapshow Window", *c.SnapshotWindow)
```

Comment:
we probably don't want to leave these in, but it's bizarre.. these lines panic for me if I don't specify a snapshot window. Does this config work for you? 

```hcl
provider "aws" {
  region = "us-west-2"
}
resource "aws_vpc" "default" {
  cidr_block = "10.250.0.0/16"
  tags {
    Name = "ec-sub-test"
  }
}

resource "aws_subnet" "private" {
  vpc_id = "${aws_vpc.default.id}"
  cidr_block = "10.250.3.0/24"
}

resource "aws_subnet" "private2" {
  vpc_id = "${aws_vpc.default.id}"
  cidr_block = "10.250.2.0/24"
}

resource "aws_elasticache_subnet_group" "test" {
  name        = "test"
  description = "Test Redis"
  subnet_ids = [
    "${aws_subnet.private.*.id}", 
    "${aws_subnet.private2.id}",
  ]
}

resource "aws_elasticache_cluster" "test" {
  cluster_id = "snapshot-tests"
  subnet_group_name = "${aws_elasticache_subnet_group.test.name}"
  num_cache_nodes = 1
  node_type = "cache.t2.micro"
  engine = "redis"
  port = 6379

  #snapshot_retention_limit = 1
  #snapshot_window = "05:00-09:00"
}
```